demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -1453,6 +1439,46 @@ export class DemoMeetingApp
     this.initContentShareDropDownItems();
   }
 
+  async setAndGetEventReporter(configuration: MeetingSessionConfiguration): Promise<EventReporter> {
+    let eventReporter: EventReporter;
+    const ingestionURL = configuration.urls.eventIngestionURL;
+    if (ingestionURL) {
```

Comment:
we can just do a if (!ingestionURL || !this.enableEventReporting) return to reduce the nested if below.

Reply:
Cool, makes sense, I still need to make decision for both if else for enableEventReporting but returning when ingestionURL is null avoids the nested if. Thanks! Have updated the code.